src/App.tsx

```diff
@@ -1,26 +1,45 @@
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import { Loader, TodoFilter, TodoList, TodoModal } from './components';
+import React, { useEffect, useState } from 'react';
+import { getTodos } from './api';
+import { useAppDispatch, useAppSelector } from './app/hooks';
+import { selectCurrentTodo, setTodos } from './features';
 
-export const App = () => (
-  <>
-    <div className="section">
-      <div className="container">
-        <div className="box">
-          <h1 className="title">Todos:</h1>
+export const App = () => {
+  const [isloading, setIsLoading] = useState(false);
```

Comment:
The state variable `isloading` should be named `isLoading` to follow camelCase naming conventions, which is standard in JavaScript and TypeScript for variable names.